src/components/Ui/Avatar.vue

```diff
@@ -28,7 +28,10 @@ watch(
 </script>
 
 <template>
-  <span class="flex shrink-0 items-center justify-center">
+  <span
+    class="flex shrink-0 items-center justify-center"
+    :style="{ width: `${parseInt(size) || 22}px` }"
```

Comment:
Btw the shrinking in Firefox looks more like a flex-box issue. But fixing that would involve the parent elements I guess. So... that also does the trick. :D